src/components/Link/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { NavigationActionType, type View } from '@/types'
+import { useNavigation } from '@/hooks'
+import { v4 as uuid } from 'uuid'
+
+interface LinkProps {
+  children: string
+  to: View
+  props?: Record<string, unknown>
+}
+
+export const Link = ({ children, to, props }: LinkProps): JSX.Element => {
+  const { state, dispatch } = useNavigation()
+  const id = uuid()
+  const linkItem = state.registry.get(to)
+  return (
+    <a
+      className='p-1 hover:font-bold'
+      href={`${linkItem.metadata.path}?id=${id}`}
+      onClick={(event) => {
+        event.preventDefault()
```

Comment:
It is already possible to context click and choose "Open in new tab" in the context menu. So it should allow for default behaviour when using modifier key (i.e also open in new tab).

```
if (event.ctrlKey || event.metaKey) {
  return true
}
```